ontology/yaml/resources/ELECTRICAL/entity_types/BATT.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BATT_GEN:
```

Comment:
nit: would suggest calling this BATT_STD or BATT_STANDARD (see precedent for standard types in GM, SAFETY)

Reply:
fixed